src/Illuminate/Console/ConfirmableTrait.php

```diff
@@ -15,6 +15,10 @@
      */
     public function confirmToProceed($warning = 'Application In Production!', $callback = null)
     {
+        if (! $this->input->isInteractive()) {
+            return true;
```

Comment:
Right now, the `confirm()` has a default value: `false`, so it will fill in false and continue (which in the case of confirmableTrait ends in 'command is aborted'.
Not sure how normal questions are handled.